pkg/notify/elasticsearch.go

```diff
@@ -112,7 +113,7 @@ type index struct {
 
 func (e *ElasticSearch) flushIndex(ctx context.Context, event interface{}) error {
 	// Create index if not exists
-	exists, err := e.ELSClient.IndexExists(e.Index).Do(ctx)
+	exists, err := e.ELSClient.IndexExists(e.Index + "-" + time.Now().Format(indexSuffixFormat)).Do(ctx)
```

Comment:
`e.Index + "-" + time.Now().Format(indexSuffixFormat)` This is repeated at several places. Can we make it a part of struct and use that?

Reply:
fixed